src/main/java/io/cdap/plugin/salesforce/plugin/source/batch/util/SalesforceSourceConstants.java

```diff
@@ -35,17 +35,15 @@
   public static final String PROPERTY_PK_CHUNK_ENABLE_NAME = "enablePKChunk";
   public static final String PROPERTY_CHUNK_SIZE_NAME = "chunkSize";
   public static final String PROPERTY_PARENT_NAME = "parent";
+  public static final String PROPERTY_QUERY_SPLITS = "querySplits";
```

Comment:
Nit: name this similar to other config params passed to the input format. Something like mapred.salesforce.input.query.split